appear/src/main/java/com/arcbees/gquery/appear/client/Appear.java

```diff
@@ -42,13 +47,158 @@ public Appear init(GQuery gq) {
     private static final String DISAPPEAR_EVENT = "disappear";
     private static final String APPEAR_EVENT = "appear";
     private static final Map<Element, AppearImpl> impls = new HashMap<>();
+    private static final Function ON_SCROLL_FUNCTION = new Function() {
+        @Override
+        public void f() {
+            hasScrolled = true;
+        }
+    };
 
     private static HandlerRegistration handlerRegistration;
+    private static boolean useMutationObserver = true;
+    private static boolean hasScrolled;
+    private static boolean hasResized;
+    private static boolean hasDomMutated;
+    private static Observe mutationHandler;
+
+    /**
+     * @param useMutationObserver Set to true to use the MutationObserver (default: true)
+     * @see <a href="http://www.w3.org/TR/dom/#interface-mutationobserver">W3C DOM4 MutationObserver</a>
+     */
+    public static void setUseMutationObserver(boolean useMutationObserver) {
+        com.arcbees.gquery.appear.client.Appear.useMutationObserver = useMutationObserver;
+        if (handlerRegistration != null) {
+            mutationHandler = createMutationHandler();
+        }
+    }
+
+    /**
+     * Force checking the appearance status of all bound elements
+     */
+    public static void checkElementsAppearance() {
```

Comment:
I will not add this static method. It's not really the GQuery way to do things. People can select their element and call $(mySelector). checkElementsAppearance();